CHANGELOG.md

```diff
@@ -18,10 +18,15 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
 
+- Added support for displaying a configurable management API URL for the user
```

Comment:
- Now displaying Managment API URL in dashboard. Displayed URL is also overridable.